addons/mail/static/src/core/common/message_reaction_menu.xml

```diff
@@ -6,14 +6,14 @@
             <div class="d-flex h-100" t-on-keydown="onKeydown" t-ref="root">
                 <div class="d-flex overflow-auto flex-column bg-100 p-2 h-100 border-end">
                     <t t-foreach="props.message.reactions" t-as="reaction" t-key="reaction.content">
-                        <button class="btn p-1 rounded-2 mx-2 py-0 d-flex align-items-center" t-att-class="{ 'bg-200 border-primary': reaction.eq(state.reaction) }" t-att-title="getEmojiShortcode(reaction)" t-on-click="() => state.reaction = reaction">
+                        <button class="btn p-1 rounded-2 mx-2 py-0 d-flex align-items-center" t-att-class="{ 'bg-200 border-primary': reaction.eq(this.state.reaction) }" t-att-title="getEmojiShortcode(reaction)" t-on-click="() => state.reaction = reaction">
```

Comment:
`this` is implicit for immediate code evaluation, i.e. everything that is not a function.
So `reaction.eq(this.state.reaction)` can just be `reaction.eq(state.reaction)`
I'm impressed that `t-on-click` is an arrow function without `this`... I thought we had to provide `this` explicitly. Probably `t-on-click` handle implicit `ctx` correctly while this won't work if you pass arrow function as a prop?